coinbaseadvanced/models/orders.py

```diff
@@ -189,7 +189,7 @@ def __init__(self, market_market_ioc: dict = None, limit_limit_gtc: dict = None,
             **stop_limit_stop_limit_gtd) if stop_limit_stop_limit_gtd is not None else None
 
 
-class Order:
+class Order(dict):
```

Comment:
I don't think we need this but let me know if you think I'm wrong. I'm gonna merge as-is and remove it as part of my follow-up.

Reply:
It isn't needed for this PR.